analysis_plan.md

```diff
@@ -0,0 +1,17 @@
+# First analysis plan
+
+This is the suggested analysis plan for the outlier detection.
+
+## Reading we need to do
```

Comment:
#5 we could maybe make this heading more formal?

Reply:
Yes, good idea - let's put the discussion in the issue.